### ISSUE TITLE
About Task 2

### ISSUE THREAD

**Issue by tirthbha** [ubsuny/23-Homework5G4#32]
For the task 2, the task is only to create graphs for the windowing, filtering and padding or something else?

**Comment by sharmistharanit** [comment]
Yes you are right. But we have to plot both the "raw" and "cleaned" spectra in the time domain. That means we have to take inverse transform.
